PostFreeSurfer/scripts/FreeSurfer2CaretConvertAndRegisterNonlinear.sh

```diff
@@ -414,44 +414,15 @@ for Hemisphere in L R ; do
 
     #If desired, run MSMSulc folding-based registration to FS_LR initialized with FS affine
     if [ ${RegName} = "MSMSulc" ] ; then
-        #Calculate Affine Transform and Apply
-        if [ ! -e "$AtlasSpaceFolder"/"$NativeFolder"/MSMSulc ] ; then
-            mkdir "$AtlasSpaceFolder"/"$NativeFolder"/MSMSulc
-        fi
+        
         ${CARET7DIR}/wb_command -surface-affine-regression "$AtlasSpaceFolder"/"$NativeFolder"/${Subject}.${Hemisphere}.sphere.native.surf.gii "$AtlasSpaceFolder"/"$NativeFolder"/${Subject}.${Hemisphere}.sphere.reg.reg_LR.native.surf.gii "$AtlasSpaceFolder"/"$NativeFolder"/MSMSulc/${Hemisphere}.mat
         ${CARET7DIR}/wb_command -surface-apply-affine "$AtlasSpaceFolder"/"$NativeFolder"/${Subject}.${Hemisphere}.sphere.native.surf.gii "$AtlasSpaceFolder"/"$NativeFolder"/MSMSulc/${Hemisphere}.mat "$AtlasSpaceFolder"/"$NativeFolder"/MSMSulc/${Hemisphere}.sphere_rot.surf.gii
         ${CARET7DIR}/wb_command -surface-modify-sphere "$AtlasSpaceFolder"/"$NativeFolder"/MSMSulc/${Hemisphere}.sphere_rot.surf.gii 100 "$AtlasSpaceFolder"/"$NativeFolder"/MSMSulc/${Hemisphere}.sphere_rot.surf.gii
         cp "$AtlasSpaceFolder"/"$NativeFolder"/MSMSulc/${Hemisphere}.sphere_rot.surf.gii "$AtlasSpaceFolder"/"$NativeFolder"/${Subject}.${Hemisphere}.sphere.rot.native.surf.gii
```

Comment:
We may want to move this code to an unconditional location (currently, it is inside an "if MSMSulc").  We could even consider having the MSMSulc script expect to find it in the `"$AtlasSpaceFolder"/"$NativeFolder"/` sphere.rot location, instead of already existing in an MSMSulc folder.

As it stands, if someone initially runs the pipelines with "FS" instead of "MSMSulc" (not recommended, results in a warning, but looks like it will proceed), and then runs MSMSulc.sh by itself, the rotated sphere file won't exist.

Reply:
I think it would be reasonable to run the sphere rotation on any registration option.